### DISCUSSION TITLE
add dragging file to filesystem feature to chromebooks

### DISCUSSION THREAD

**Discussion by eggosarethebest** [godotengine/godot-proposals#6299] [Platforms]
I recently got into game development and only have a chromebook without linux so I heard about godot on the playstore so I downloaded it and tried to follow tutorials but when I tried to drag a file into godot and it didn't work 

[Screen recording 2023-02-15 9.24.50 PM.webm](https://user-images.githubusercontent.com/120989935/219260634-a9b34d2e-008a-4e8f-ad7c-d389b6a76835.webm)

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/6168.
